### DISCUSSION TITLE
How to display my widget data in left bottom corner of taskbar on Windows 11?

### DISCUSSION THREAD

**Discussion by steam3d** [microsoft/WindowsAppSDK#3950] [Q&A]
I want to display data of my widget here:
![image](https://github.com/microsoft/WindowsAppSDK/assets/30986874/d6cefb4a-376c-4677-8aae-d36569080c1e)

I could not find any info about it.

**Comment by Yewzir** [comment]
@steam3d 

Since that looks like the Weather widget, you should click Start, and type _weather_ in your operating system's display language, then pin the application to Start.

![image](https://github.com/user-attachments/assets/84bc18c4-4469-4874-893c-7acb48c87f3d)

Click Start once again, and drag the newly added widget to the position of your choice.

![image](https://github.com/user-attachments/assets/c9c5ed68-455e-4686-8d58-34539043fadc)

That is pretty much all there is to it.

Kind regards
Yewzir

**Comment by steam3d** [comment]
@Yewzir I meant the Windows 11 widgets. You described the Windows 10 live tiles.

**Reply by Yewzir** [reply]
Honest mistake. Although, I've had Windows 11 before, and I think its customization of the Start menu is kind of limited to previous editions of the operating system. With some reordering, you could put it all the way to the left on the taskbar, though?

**Reply by steam3d** [reply]
@Yewzir I have no idea why you are talking about the Start menu when I asked a question about Windows 11 widgets. Here is the screenshot of Windows 11. On the Widgets board, you can see the weather widget. This widget has the ability to deliver weather status to the small widget in the taskbar. I want to do the same with my widget.
![5vk6fqa9zzQ6iJQ7WKGNVn-1920-80 jpg](https://github.com/user-attachments/assets/82c8ea6f-b37f-469c-a388-c12da63d49b4)

Here is official documentation:
https://learn.microsoft.com/en-us/windows/apps/develop/widgets/implement-widget-provider-cs

It does not mention how to achieve this.

**Reply by Yewzir** [reply]
Do you realize your title refers to the start panel to begin with?

> How to display my widget data in left bottom corner of start panel? #3950

**Reply by steam3d** [reply]
@Yewzir my bad. I renamed it.

**Reply by Yewzir** [reply]
That's all good.

1. Right-click on the taskbar and select **Taskbar settings**.
2. In the Taskbar settings window, scroll down to the **Taskbar items** section.
3. Ensure that the **Widgets** toggle is turned **on**.
4. Click on the **Widgets icon** on the taskbar to open the **widgets panel**.
5. In the widgets panel, click on the **three dots** in the top-right corner of the weather widget.
6. Select **Customize widget** to set your preferences for the weather widget.
7. Once customized, close the widgets panel.
8. To move the widgets panel to the bottom left, you may need to rearrange other taskbar items. **Drag** the widgets icon towards the bottom left corner of the taskbar. If your taskbar items are centered, it should be no problem. Otherwise, you will have to go back to the Taskbar settings and ensure your Taskbar items are positioned anywhere but on the left.

Kind regards
Yewzir

**Reply by michalleptuch** [reply]
@steam3d you can't. I saw the release notes of 1.6-experimental2 and there is a sentence:
> New extensions enable Widget Providers to provide Widgets with web content and announcements for Widgets.

I'm not sure if announcements are exactly what you want but maybe this is a hint.
